Connor

cjlysek opened the pull request:
Updated Session.js to display the mentor questions/responses at the bottom of the logged answers. After testing through what should be displayed and postman, the questionIDs matched what the session was given and the responses as well.

cjlysek committed:
Update Session.js

Added in student's mentor responses now that the API is working properly. Get mentor question is not functional yet since the API is not working, so this will be updated before merging

cjlysek committed:
Update Session.js

Fixed the state when the modal is open to clear the previous mentorResponses so they are accurate, and added in a noQuesMsg so if there are logged answers but not any mentor responses, it will state that there are none found.

(Still waiting to fix the /question API)

cjlysek committed:
Update Session.js

Finished updating getMentorQuestion so the questions match and show on the sessions page with the correct response. After testing it appears to be working properly.

Naton-1 commented:
Before merging this:
- [ ] Add thin gray line as a separator between each row of Mentor Questions/Answers
- [ ] "Mentor Questions" -> "Mentor Questions/Responses"

cjlysek committed:
Update Session.js

Changed Mentor Questions to Mentor Questions/Responses and added in a line border below each question/response to separate them

smarosi1 reviewed (approved):
Looks good
